exist-core/src/main/java/org/exist/repo/Deployment.java

```diff
@@ -392,6 +397,12 @@ private void checkProcessorVersion(final PackageLoader.Version version) throws P
                     targetCollection = XmldbURI.SYSTEM.append("repo/" + pkgColl);
                 }
 
+                // skip trigger execution if demanded in repo.xml
+                final Optional<ElementImpl> skipTriggers = findElement(repoXML, SKIP_TRIGGERS_ELEMENT);
+                if(skipTriggers.isPresent()) {
```

Comment:
Should we perhaps check for `<skipTriggers>true</skipTriggers>` seems more natural

Reply:
Hm, to me the empty but present `skip-triggers` element feels more natural, but I don't mind. Whatever exist-core prefers to see.

Reply:
Reading through the [repo documentation](http://exist-db.org/exist/apps/doc/repo.xml#sect-repoxml), what about
`<triggers skip="{true/false}" />` similar to the permissions setting?

Reply:
@line-o yet another option, I really don't mind the representation. Shall we vote or who's the dictator?

Reply:
No dictators here, just consensus.

Reply:
If I have to sell my proposal: `<triggers />` is easier to extend. The attribute name `execute-on-install` is easy to grasp. `on|off` or `true|false` should not matter, but depends on the current implementation what counts as a boolean.

Reply:
@chakl @line-o I would suggest `<triggers skip-on-install="true"/>`

Reply:
Also what happens with uninstall of the app? Are triggers notified?